src/org/openstreetmap/josm/gui/dialogs/SearchDialog.java

```diff
@@ -231,7 +232,7 @@ public boolean isValid() {
          * selected preset by the user. Every query is of the form ' group/sub-group/.../presetName'
          * if the corresponding group of the preset exists, otherwise it is simply ' presetName'.
          */
-        selector = new TaggingPresetSelector(false, false, false);
+        selector = new TaggingPresetSelector(PresetSearchFilter.values());
```

Comment:
This would be the equivalent of `new TaggingPresetSelector(true, true, true)`, which I don't _think_ you wanted. You probably wanted to just do `new TaggingPresetSelector()`.